webscrapers/covents_json.json

```diff
@@ -0,0 +1,142 @@
+{
+  "1": {
+    "id": "1",
+    "event_name": "Zayn Malik Party",
+    "event_image": "https://img.evbuc.com/https%3A%2F%2Fcdn.evbuc.com%2Fimages%2F100189414%2F444571810080%2F1%2Foriginal.20200505-023120?w=500&auto=format%2Ccompress&q=75&sharp=10&rect=0%2C125%2C500%2C250&s=b8e2f287f2ff81f05f5686af7315a584",
+    "event_date_time": "Sun, May 31, 2020 6:00 PM MDT",
```

Comment:
This looks great!, we can break the date and time up for the FE as we store the values in the DB.  Awesome job, Dan!

Reply:
^ that would be great, Kelly! Agreed, this looks awesome!